doc/api/crypto.md

```diff
@@ -2342,12 +2347,17 @@ be passed instead of a public key.
 <!-- YAML
 added: v0.11.14
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/28335
+    description: The `oaepHash` option was added.
   - version: v11.6.0
     pr-url: https://github.com/nodejs/node/pull/24234
     description: This function now supports key objects.
 -->
 * `key` {Object | string | Buffer | KeyObject}
   - `key` {string | Buffer | KeyObject} A PEM encoded public or private key.
+  - `oaepHash` {string} The hash function to use for OAEP padding.
```

Comment:
Could we just call this `hash` or perhaps `paddingHash`, so when the next padding mode comes along we don't need a parameter per padding mode?

Reply:
I considered that and I am glad you brought it up. My main concern is a clash with key options, to be honest. Also, other padding modes might also accept a hash function as a parameter, but maybe with entirely different semantics. Anway, I am not sure either.

Reply:
I see your point. Its hard to think of a better name, I'm ok with this one.

Reply:
I'm ok with this one as well, but not entirely happy to be honest. OpenSSL does have specific names as well, though, so it might not be the worst idea ever.